llvm/lib/IR/Metadata.cpp

```diff
@@ -1302,6 +1302,26 @@ static void addRange(SmallVectorImpl<ConstantInt *> &EndPoints,
   EndPoints.push_back(High);
 }
 
+MDNode *MDNode::getMergedCalleeTypeMetadata(LLVMContext &Ctx, MDNode *A,
+                                            MDNode *B) {
+  SmallVector<Metadata *, 8> AB;
+  SmallSet<Metadata *, 8> MergedCallees;
+  auto AddUniqueCallees = [&](llvm::MDNode *N) {
+    if (!N)
+      return;
+    for (const MDOperand &Op : N->operands()) {
+      Metadata *MD = Op.get();
+      if (!MergedCallees.contains(MD)) {
+        MergedCallees.insert(MD);
+        AB.push_back(MD);
```

Comment:
`SmallSet::inesert(()` returns `pair<iterator, bool>`, so you don't need to use `contains()`
```suggestion
      if ( MergedCallees.contains(MD).second) {
        AB.push_back(MD);
```